### PR TITLE
Modularized configs and readable code structure

### PR DESCRIPTION
**Pull request by ramsainanduri**
- created custom configs for each subworkflow along with the module specific
  - added separate variables for frequency cutoff for variant calling in the hopper config
  - removed cnvkit old process from  modules/local/cnvkit and all the respective files
  - removed ASCAT module
  - removed process 'ASCAT2SCARHRD, ASCAT2OVAHRDSCAR' from modules/local/hrdsw
  - removed stub block for Check csv (because it is not required)
  - GATK CNVs related process are now executed on a condition when gatk_cnv is set to true

### CODE REVIEW

**Comment by ramsainanduri** [comment]
All profiles were executed without errors

**Comment by ramsainanduri** [comment]
Git ready, folks, this commit is so massive, my apologies for the coding rampage.

File: configs/modules/annotate_cnv.config
Comment (ramsainanduri) [inline configs/modules/annotate_cnv.config:1]: should remove this file

File: configs/modules/annotate_snv.config
Comment (ramsainanduri) [inline configs/modules/annotate_snv.config:1]: this file is removed

File: configs/modules/base.config
Comment (ramsainanduri) [inline configs/modules/base.config:1]: Added multiple combinations of withLabel configs for memory, CPUs, and time for future use

File: configs/modules/create_cnv_pon.config
Comment (ramsainanduri) [inline configs/modules/create_cnv_pon.config:1]: this file is removed

File: modules/local/ASCAT/main.nf
Comment (ramsainanduri) [inline modules/local/ASCAT/main.nf:1]: removed this module

File: bin/vcf2.pm
Comment (ViktorHy) [inline bin/vcf2.pm:1]: was this necessary? Have you tried the code? normally a perl-module does not need a shebang

Comment (ramsainanduri) [reply bin/vcf2.pm:1]: I tested the pipeline for all four profiles on real samples. It worked well. Sorry I didn't know that perl modules does not need a shebang

**Review by ViktorHy — COMMENTED** [review]
are all entry-points tested? are all scripts that have had changes tested?

**Comment by ramsainanduri** [comment]
Only SSP entry point is tested with four profiles

**Comment by ViktorHy** [comment]
OK, we still need to fix the other entry-points so that they use similar input csvs as the main entrypoint. We could merge this now

**Review by ViktorHy — APPROVED** [review]
tested the code, besides some python-scripts now requiring chmod +x to be executable it seems to work just fine.

**Comment by ramsainanduri** [comment]
Merging it

**Pull request merged**